virtual-repeater.js

```diff
@@ -222,8 +224,17 @@ export const Repeats = Superclass => class extends Superclass {
    */
   async _measureChildren() {
     if (this._ordered.length > 0) {
+      // Grab the children to be measured. We do this
+      // synchronously as _toMeasure relies on _prevFirst and _prevLast.
       const {indices, children} = this._toMeasure;
+      // We debounce the measurement of a microtask.
+      const id = ++this._measuringId;
       await Promise.resolve();
```

Comment:
@graynorton do you have suggestions on how to "justify" this microtask?
I was thinking this was added to give time to classes/styles to be applied, but by this time we already did call _newChild and _updateChild, so the user has had already the time to update styles. Also, getBoundingClientRect() forces relayout.

Reply:
Actually, this helps play nice with browser timing and avoids forcing layout, while still being executed before the next frame. Here a comparison of the demo with & without microtask wait:

 sync execution | await microtask 
---| ---
![no-wait](https://user-images.githubusercontent.com/6173664/39655904-3deb1626-4fb1-11e8-892a-b826773d0c93.gif) |  ![wait](https://user-images.githubusercontent.com/6173664/39655906-46769c34-4fb1-11e8-96e9-6437840e174f.gif)


So, we keep it!